### PR TITLE
Add example for long context mistral model

### PR DESCRIPTION
**Pull request by gvspraveen**

### CODE REVIEW

File: templates/endpoints/models/mistral/NousResearch--Yarn-Mistral-7b-128k_a10g_tp1.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+deployment_config:
```
Comment (akshay-anyscale) [inline templates/endpoints/models/mistral/NousResearch--Yarn-Mistral-7b-128k_a10g_tp1.yaml:1]: file should be named tp8

Comment (gvspraveen) [reply templates/endpoints/models/mistral/NousResearch--Yarn-Mistral-7b-128k_a10g_tp1.yaml]: done

File: configs/endpoints/aws.yaml [hunk 2]
```diff
@@ -61,6 +61,19 @@ worker_node_types:
   max_workers: 100
   use_spot: true
   fallback_to_ondemand: true
+- name: gpu-worker-a10g-8
```
Comment (akshay-anyscale) [inline configs/endpoints/aws.yaml:64]: let's add one for GCP too?

**Comment by akshay-anyscale** [comment]
LGTM, let's also copy over to v2

**Review by akshay-anyscale — APPROVED** [review]

**Pull request merged**